test/support/blockchain_sim.ex

```diff
@@ -152,7 +152,7 @@ defmodule AeMdwWeb.BlockchainSim do
 
     {_prev_hash, mock_blocks, mock_txs, _max_height} =
       blocks
-      |> Enum.reduce({@genesis_prev_hash, %{}, %{}, @initial_height}, fn
+      |> Enum.reduce({@genesis_prev_hash, %{}, [], @initial_height}, fn
```

Comment:
Switching them to `Keyword` instead of `Map` makes the order deterministic (some tests rely on that)